diff --git a/src/tests/utility.hpp b/src/tests/utility.hpp
index 92723b97..ceb33d9f 100644
--- a/src/tests/utility.hpp
+++ b/src/tests/utility.hpp
@@ -32,6 +32,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <cstdlib>
 #include <iostream>
 #include <fstream>
+#include <cstdint>
 
 constexpr char hexmap[] = "0123456789abcdef";
 inline void outputHex(std::ostream& os, const char* data, int length) {
