diff --git a/src/lang.h b/src/lang.h
index e6236c7..67b1cc0 100644
--- a/src/lang.h
+++ b/src/lang.h
@@ -40,7 +40,7 @@
  *                                                                      *
  ************************************************************************/
 #ifndef LANG  /* Allow for definition on compiler command line */
-#define LANG ENGLISH
+#define LANG POLISH
 #endif
 
 /* Pick up the appropriate header file */
